src/main/java/com/jcabi/github/RtNotifications.java

```diff
@@ -52,4 +52,26 @@
     public Notification get(final int number) {
         return new RtNotification(number);
     }
+
+    /**
+     * Marks all notifications on this repository as read.
+     * @see <a href="https://developer.github.com/v3/activity/notifications/#mark-notifications-as-read-in-a-repository">Mark notifications as read in a repository</a>
+     * @todo Implement this method
+     */
+    @Override
+    public void markAsRead() {
+        // Will be implemented later
+    }
+
+    /**
+     * Get thread data.
+     * @see <a href="https://developer.github.com/v3/activity/notifications/#view-a-single-thread">View a single thread</a>
+     * @param number Thread ID.
```

Comment:
Do you mean `// Will be implemented later`? I added it because otherwise Checkstyle or PMD complains about an empty method.